wirelength_analyzer/README.md

```diff
@@ -0,0 +1,115 @@
+Routing path for Critical Path
+Wirelength: 663
+Segment | Running |
+Length  |  Total  | Segment Name
+--------+---------+-----------------------------------------
+        |        0| cell    u_calc/dropSpin/scattererReflector/squareRoot1_6/res__3_q_reg[40]
+       0|         | belPin  SLICE_X71Y84 AFF Q (start of net: u_calc/dropSpin/scattererReflector/squareRoot1_6/res__3_q[40])
+       0|         | belPin  SLICE_X71Y84 AQ AQ
+       0|         | sitePin SLICE_X71Y84 AQ
+       0|         | pip     INT_X46Y84 LOGIC_OUTS_E14 INT_NODE_SDQ_17_INT_OUT0 True False
+       5|         | pip     INT_X46Y84 INT_NODE_SDQ_17_INT_OUT0 EE2_E_BEG3 True False
+       0|         | pip     INT_X47Y84 EE2_E_END3 INT_NODE_SDQ_14_INT_OUT0 True False
+       3|         | pip     INT_X47Y84 INT_NODE_SDQ_14_INT_OUT0 NN2_E_BEG2 True False
+       0|         | pip     INT_X47Y86 NN2_E_END2 INT_NODE_SDQ_13_INT_OUT0 True False
+      10|         | pip     INT_X47Y86 INT_NODE_SDQ_13_INT_OUT0 WW4_E_BEG2 True False
+      12|         | pip     INT_X45Y86 WW4_E_END2 NN12_BEG3 True False
+       0|         | pip     INT_X45Y98 NN12_END3 INT_NODE_SDQ_12_INT_OUT0 True False
+       1|         | pip     INT_X45Y98 INT_NODE_SDQ_12_INT_OUT0 NN1_E_BEG2 True False
+       0|         | pip     INT_X45Y99 NN1_E_END2 INT_NODE_SDQ_8_INT_OUT0 True False
+       1|         | pip     INT_X45Y99 INT_NODE_SDQ_8_INT_OUT0 EE1_E_BEG1 True False
+       0|         | pip     INT_X46Y99 EE1_E_END1 INT_NODE_SDQ_53_INT_OUT0 True False
+       1|         | pip     INT_X46Y99 INT_NODE_SDQ_53_INT_OUT0 SS1_W_BEG1 True False
+       0|         | pip     INT_X46Y98 SS1_W_END1 INT_NODE_SDQ_50_INT_OUT0 True False
+       0|         | pip     INT_X46Y98 INT_NODE_SDQ_50_INT_OUT0 INT_INT_SDQ_60_INT_OUT1 True False
+       0|         | pip     INT_X46Y98 INT_INT_SDQ_60_INT_OUT1 INT_NODE_SDQ_1_INT_OUT0 True False
+       3|         | pip     INT_X46Y98 INT_NODE_SDQ_1_INT_OUT0 SS2_E_BEG0 True False
+       0|         | pip     INT_X46Y96 SS2_E_END0 INT_NODE_SDQ_22_INT_OUT0 True False
+       5|         | pip     INT_X46Y96 INT_NODE_SDQ_22_INT_OUT0 SS4_E_BEG0 True False
+       0|         | pip     INT_X46Y91 SS4_E_BLS_0_FT0 SDQNODE_E_93_FT0 True False
+       5|         | pip     INT_X46Y92 SDQNODE_E_BLN_93_FT1 WW2_E_BEG0 True False
+       0|         | pip     INT_X45Y91 WW2_E_BLS_0_FT0 INODE_E_62_FT0 True False
+       0|         | pip     INT_X45Y92 INODE_E_BLN_62_FT1 BOUNCE_E_0_FT1 True False
+       0|         | sitePin SLICE_X69Y92 AX
+       0|         | belPin  SLICE_X69Y92 AX AX
+       0|         | belPin  SLICE_X69Y92 CARRY8 AX
+        |       46| cell    u_calc/dropSpin/scattererReflector/squareRoot1_6/op__7_q_reg[63]_i_376__0
+<Truncated for brevity>
+```
+
+As seen above at this level of verbosity the `...` is replaced with the detailed
+path taken between two cells. The final verbosity level, level 0, prints the
+total wirelength and omits all other output. The second mode,
```

Comment:
Perhaps consider a paragraph break between explaining verbosity level 0 and `longest-single-net` mode.

Reply:
added a paragraph break: https://github.com/Xilinx/fpga24_routing_contest/pull/25/commits/3b297010d0544da87bdf7dd0608dcd0c62404973